tests/integrations/load-scripts-on-main-thread/index.html

```diff
@@ -91,6 +101,18 @@ <h1>Load scripts on main thread 🎉</h1>
         })()
       </script>
     </li>
+    <li>
+      <strong>Script Type:</strong>
+      <code id='regexTestScript'></code>
+      <script type='text/partytown'>
+        (() => {
+          const scriptElement = document.getElementById('testScript');
+          const codeElement = document.getElementById('testScriptType');
```

Comment:
```suggestion
      <code id='regexTestScriptType'></code>
      <script type='text/partytown'>
        (() => {
          const scriptElement = document.getElementById('regexTestScript');
          const codeElement = document.getElementById('regexTestScriptType');
```
This needs a fix I guess